docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -253,7 +253,7 @@ To enable this feature, an application needs to target .NET Framework 4.8 or opt
 </configuration>
 ```
 
-Once enabled, all controls that contain a tooltip display it once the control receives keyboard focus. The tooltip can be dismissed over time or when the keyboard focus changes. Users can also dismiss the tooltip manually by using a new keyboard shortcut, Ctrl + Shift + F10. Once the tooltip has been dismissed it can be displayed again by using the same keyboard shortcut.
+Once enabled, all controls that contain a tooltip display it once the control receives keyboard focus. The tooltip can be dismissed over time or when the keyboard focus changes. Users can also dismiss the tooltip manually by using a new keyboard shortcut, **Ctrl** + **Shift** + **F10**. Once the tooltip has been dismissed, it can be displayed again by using the same keyboard shortcut.
```

Comment:
Keyboard shortcuts aren't bold per this CG article https://review.docs.microsoft.com/en-us/help/contribute/text-formatting-guidelines?branch=main#keys-and-keyboard-shortcuts

Reply:
Thanks. I guess if you use them in procedural instructions, then they are to be bolded according to https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29014. But these are not instructions.